### PR TITLE
COP-3573: Rename the sGMR service

### PR DESCRIPTION
**Pull request by ajames14**
## Description
This branch updates the service to ‘Submit a Pleasure Craft Report’. It also changes any reference to "Notifications" to "Reports" throughout the app and updates the tests for this change. 

To test:
- run the app locally. You will see the name of the service changed on Landing page, Nav bar, Dashboard, submission page, etc. See code for further details. 
- Run tests to check for these updates.

## Developer Checklist

\* Required

- [x] Up-to-date with master branch? \*

- [ ] Does it have tests?

- [x] Pull request URL linked to JIRA ticket? \*

- [ ] All reviewer comments resolved/answered? \*

Ensure you delete the branch once the Pull Request is merged.

### CODE REVIEW

File: cypress/support/commands.js [hunk 1]
```diff
@@ -190,10 +190,10 @@ Cypress.Commands.add('selectCheckbox', (option) => {
     .should('be.checked');
 });
 
-Cypress.Commands.add('checkNotifications', (type, numberOfNotification) => {
+Cypress.Commands.add('checkReports', (type, numberOfReport) => {
```
Comment (rafenden) [inline cypress/support/commands.js:193]: It's a good idea to use plural here.

File: src/components/Dashboard.jsx [hunk 2]
```diff
@@ -51,8 +51,8 @@ const Dashboard = (pageData) => {
     <div className="govuk-width-container">
       <div className="govuk-grid-row">
         <div className="govuk-grid-column-one-half">
-          <h2 className="govuk-heading-m">Create a new notification</h2>
-          <p className="govuk-body">You can use the online form to create and submit an Advance Voyage Notification to UK Border Force.</p>
+          <h2 className="govuk-heading-m">Create a new report</h2>
+          <p className="govuk-body">You can use this online form to create and submit a pleasure craft report to UK Border Force, notifying them of your travel plans.</p>
```
Comment (rafenden) [inline src/components/Dashboard.jsx:55]: Shall we be consistent and use `Pleasure Craft Report` in the same form as [the legislation](https://www.legislation.gov.uk/uksi/1996/1406/made) and the rest of the places in the app?

```suggestion
          <p className="govuk-body">You can use this online form to create and submit a Pleasure Craft Report to UK Border Force, notifying them of your travel plans.</p>
```

File: src/components/PrivacyCookiePolicy.jsx [hunk 3]
```diff
@@ -6,16 +6,16 @@ const PrivacyCookiePolicy = () => {
       <main className="govuk-main-wrapper govuk-main-wrapper--auto-spacing" id="main-content" role="main">
         <div className="govuk-grid-row">
           <div className="govuk-width-container">
-            <h1 className="govuk-heading-l">Privacy notice for General Maritime (Pleasure Craft) Advanced Voyage Notification</h1>
+            <h1 className="govuk-heading-l">Privacy notice for the Submit a Pleasure Craft Report</h1>
```
Comment (rafenden) [inline src/components/PrivacyCookiePolicy.jsx:9]: Are we using `Submit a Pleasure Craft Report` or `Pleasure Craft Report` as the new project name?

File: src/components/contentArray.js [hunk 4]
```diff
@@ -14,9 +14,9 @@ const contentArray = [
   {
     urlStub: '/vessels',
     pageHeading: 'Vessels',
-    pageBlurb: 'You can view, edit or delete vessels that you regularly add to Advanced Voyage Reports.',
+    pageBlurb: 'You can view, edit or delete vessels that you regularly add to Pleasure Craft Reports.',
     formIntroHeading: 'Save a new vessel',
-    formIntroBlurb: 'Save the details of vessels that you regularly submit Advanced Voyage Reports for.',
+    formIntroBlurb: 'Save the details of vessels that you regularly submit Pleasure Craft Reports for.',
```
Comment (rafenden) [inline src/components/contentArray.js:19]: Shall we use `submit Pleasure Craft Reports` or `Submit Pleasure Craft Reports`?

File: src/components/PrivacyCookiePolicy.jsx (hunk 3 above)
Comment (ajames14) [reply src/components/PrivacyCookiePolicy.jsx:9]: The new project name is "Submit a Pleasure Craft Report" - however some sentences it doesn't make sense to have the Submit grammatically.

File: src/components/contentArray.js (hunk 4 above)
Comment (p3dr0migue1) [reply src/components/contentArray.js:19]: I would say keep it consistent with the project name and therefore `Submit Pleasure Craft Reports`. But ultimately content should be reviewed by Cristina .

**Review by rafenden — APPROVED** [review]

**Pull request merged**